internal/v1/handler_blueprints_test.go

```diff
@@ -837,16 +864,124 @@ func TestHandlers_GetBlueprint(t *testing.T) {
 	}
 }
 
+func marshalAndUnmarshalPrint(t *testing.T, v interface{}) []byte {
+	data, err := json.Marshal(v)
+	require.NoError(t, err)
+
+	var dataUnmarshal interface{}
+	err = json.Unmarshal(data, &dataUnmarshal)
+	require.NoError(t, err)
+
+	data, err = json.MarshalIndent(dataUnmarshal, "", "  ")
+	require.NoError(t, err)
+
+	return data
+}
+
+func compareOutputExportBlueprint(t *testing.T, jsonResponse []byte) {
+	file, err := os.CreateTemp(t.TempDir(), "exported_blueprint_temp.*.json")
+	require.NoError(t, err)
+
+	jsonFile, err := os.Create(file.Name())
+	require.NoError(t, err)
+
+	_, err = jsonFile.Write(jsonResponse)
+	require.NoError(t, err)
+
+	exportedBlueprintFile := "../common/testdata/exported_blueprint.json"
+	fixtureData, err := os.ReadFile(exportedBlueprintFile)
+	require.NoError(t, err)
+
+	generatedData := marshalAndUnmarshalPrint(t, jsonResponse)
+	require.NoError(t, err)
+
+	expectedData := marshalAndUnmarshalPrint(t, fixtureData)
+	require.NoError(t, err)
+
+	require.JSONEq(t, string(expectedData), string(generatedData))
```

Comment:
Nitpick: If this does not print the full JSON on error, can you also print it? In case it is off in the future so it is easy to update it. Ideally nicely formatted. All you need to do is to use `assert` instead of `require` and check for error.

Reply:
Okay, it used to print in base64, but now it should be more precise.